### DISCUSSION TITLE
InputData location for jobs sumitted in "local" mode

### DISCUSSION THREAD

**Discussion by arrabito** [DIRACGrid/DIRAC#6390] [Q&A] [answered]
Dear all,

I've tried to submit jobs with InputData in local mode and I found that the following option should be set:

Operations/InputDataPolicy/InputDataModule 

So I set it to:
DIRAC.WorkloadManagementSystem.Client.DownloadInputData

However I've noticed tha Input files are downloaded in a specific subdirectory of the local job directory instead of directly in the local job directory. 

I found that there is a parameter that can be set to change the InputData directory in the argumentsDict:
https://github.com/DIRACGrid/DIRAC/blob/integration/src/DIRAC/Interfaces/API/Dirac.py#L571
however it does not seem to be a configurable option, but it can just be hardcoded in Dirac.py.

Could you please comment on that ?

Thanks
Luisa

**Comment by arrabito** [comment, marked as answer]
I found that:

DIRAC.WorkloadManagementSystem.Client.InputDataResolution

has the desired behavior of downloading input data in the job local directory.

I mark the discussion as answered.

**Reply by chaen** [reply]
I am a tad late on that, but please update the doc (https://dirac.readthedocs.io/en/rel-v7r3/AdministratorGuide/Systems/WorkloadManagement/InputDataResolution.html) if you think something is missing :-)
